jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/rs/core/headers/JAXRSClientIT.java

```diff
@@ -210,6 +210,26 @@ public void requestHeadersTest() throws Fault {
     invoke();
   }
 
+  /*
+   * @testName: requestHeadersTest
+   * 
+   * @assertion_ids: JAXRS:JAVADOC:1361; JAXRS:JAVADOC:1362;
+   * 
+   * @test_Strategy: HttpHeaders.containsHeaderString used to confirm that a given header contains
+   * a specified value with the appropriate value separator.
+   */
+  @Test
+  public void containsHeaderStringTest() throws Fault {
+    setProperty(Property.REQUEST_HEADERS,
+        "Accept:text/*, text/html, text/html;level=1, */*");
+    setProperty(Property.REQUEST_HEADERS,
+        "Content-Type:application/xml;charset=utf8");
+    setProperty(Property.REQUEST_HEADERS,
+        "Header3:value1 ** value2 **value 3");
+    setProperty(Property.REQUEST, buildRequest(Request.GET, "contains-headers"));
+    invoke();
```

Comment:
the test does not check the result, no-matter the exception being thrown on the server, the test just passes.

Reply:
Thanks.   I believe this is addressed.